### ISSUE TITLE
Change Request: Make logo larger in site header [Drupal Dev]

### ISSUE THREAD

**Issue by meglake** [NCIOCPL/cgov-digital-platform#3605]
Per feedback from client on 10/17, they would like to see a more proportional hierarchy in the header between the logo, section nav items, and megamenu items.

- [ ] Revise the site header to adjust the size of the site logo to the following:
-- Widescreen/Desktop: 50px 
-- Tablet/Mobile: 38px (logo is wrapped to two lines where necessary - for DS dev this includes: Cgov English, Cgov Spanish, DCEG, Nano, and Physics) 

**Due: ASAP week of 10/24 (this should be top priority)**

Once ticket is complete we'll review on ACSF Test (along with changes from [#615](https://github.com/NCIOCPL/ncids/issues/615)) with client to re-confirm approval on logo sizes across all breakpoints and on all microsites. 

> **ESTIMATE** TBD

### What's the expected change?
- Site logo shows in header at:
-- Widescreen/Desktop: 50px
-- Tablet 38px
-- Mobile 38px (longer logos changed to 2-line stacked, for DS dev this includes: Cgov English, Cgov Spanish, DCCPS, and DCEG)

### What's the current functionality?
- Site logo shows in header at 385x38 for all breakpoints

### What's the updated acceptance criteria?
```gherkin
```

### Related Tickets
* Design Ticket https://github.com/NCIOCPL/ncids/issues/625 
* DS Dev https://github.com/NCIOCPL/ncids/issues/626

**Comment by sarinapadilla** [comment]
Closing this ticket, as it is a duplicate of #3610.
